WordPress/Classes/Services/AccountService.m

```diff
@@ -150,50 +148,24 @@ - (void)removeDefaultWordPressComAccount
  @return a WordPress.com `WPAccount` object for the given `username`
  @see createOrUpdateWordPressComAccountWithUsername:password:authToken:
  */
-- (WPAccount *)createOrUpdateWordPressComAccountWithUsername:(NSString *)username
+- (WPAccount *)createOrUpdateAccountWithUsername:(NSString *)username
                                                    authToken:(NSString *)authToken
```

Comment:
I am sorry for the annoying comment but there is a little bit misalignment here after the method name change.